programs/vote/src/vote_instruction.rs

```diff
@@ -164,7 +170,7 @@ pub fn withdraw(
 }
 
 pub fn process_instruction(
-    _program_id: &Pubkey,
+    program_id: &Pubkey,
```

Comment:
is this used now?

Reply:
afaik, no.

Reply:
I don't see it referenced in this function...  does this compile?

Reply:
not yet